tests/specs/packages.js

```diff
@@ -0,0 +1,190 @@
+define(["dist/amd-loader"], function(AMDLoader) {
+  describe("Test suite", function() {
+    var bitloader;
+    beforeEach(function() {
+      bitloader = new AMDLoader({
+        baseUrl: "../",
+        packages: [
+          {
+            location: "tests",
+            main: "main",
+            name: "js"
+          }
+        ]
+      });
+    });
+
+    describe("When Importing a number", function() {
+      var number;
+      beforeEach(function() {
+        return bitloader.import("js/number.js")
+          .then(function (_number) {
+            console.log(arguments);
+            number = _number;
+          });
+      });
+
+      it("then the number is 3.14", function() {
+        expect(number).to.equal(3.14);
+      });
+    });
+
+    describe("When Importing an empty object", function() {
+      var result;
+      beforeEach(function() {
+        return bitloader.import("js/emptyobject.js")
+          .then(function (_result) {
+            result = _result;
+          });
+      });
+
+      it("then the result is an object", function() {
+        expect(result).to.be.an("object");
+      });
+
+      it("then the result is an object with no properties", function() {
+        expect(Object.keys(result).length).to.equal(0);
+      });
+    });
+
+    describe("When Importing two files", function() {
+      var emptyobject, emptystring;
+      beforeEach(function () {
```

Comment:
I'm thinking of using `before` for (all) of the unit tests in this file. We don't really need to run the import once for every describe, so I think it would be okay to switch to using `before`. I guess it does decrease test integrity though, since things outside the current `describe` could affect it.